core/src/modules/errors.ts

```diff
@@ -0,0 +1,27 @@
+export class InfoError extends Error {
```

Comment:
(idea, not blocking) If you wrap these in a namespace, you can then export all the Errors from `index.ts` - which would allow plugins in the future to use any of the errors:

```ts
exports namespace Errors {
  export class InfoError {}
}

```

Then in `index.ts` `export {Errors}  from '/path/to/file'`

Finally in plugins: 

```ts
import {Errors} from `@grouparoo/core`
throw new Errors.InfoError('Oh no!')
```